app/controllers/sessions_controller.rb

```diff
@@ -3,7 +3,7 @@ def new; end
 
   def create
     user = User.find_by(email: params[:session][:email].downcase)
-    if user && user.authenticate(params[:session][:password])
+    if user&.authenticate(params[:session][:password])
```

Comment:
Tại sao em sửa thế này

Reply:
rubocop nhắc sửa thế anh

Reply:
Em tìm hiểu tại sao rubocop bảo sửa thế

Reply:
em nghĩ là giúp cho cú pháp ngắn gọn hơn thôi ạ

Reply:
chưa đúng em, em tìm hiểu The Safe Navigation Operator (&.) nhé

Reply:
thì cái &. này giúp debug dễ hơn ạ, ngoại trừ nếu đối tượng là nil thì sẽ trả về nil thì các trường hợp khác như là false nó sẽ báo lỗi ạ